src/workerd/api/form-data.c++

```diff
@@ -348,8 +350,8 @@ void FormData::parse(jsg::Lock& js,
       data.reserve(query.size());
       for (auto& param: query) {
         data.add(Entry{
-          .name = kj::mv(param.name),
-          .value = kj::mv(param.value),
+          .name = js.accountedKjString(param.name),
```

Comment:
Here comes my annoying question again.  Do we intend to make FormData use DOMStrings eventually, or is the concept of DOMString irrelevant to FormData?

Reply:
Eventually maybe? I'd rather tackle that separately